ceryx/nginx/lualib/router.lua

```diff
@@ -54,3 +62,16 @@ end
 cache:set(host, res, 5)
 
 ngx.var.container_url = res
+
+local settings_key = prefix .. ":settings:" .. host
```

Comment:
Could you move the whole logic before the routing part? There's no need to check for this in two different places.